.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint --telemetry=off .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```

Comment:
- [ ] I think it would be nice if all of the comments related to code should have been put inline to codes that they are related to, in order to make it simple for the one who receives them so that they know what to change and where to change it, I'd like to suggest you put [this comment](https://github.com/hunter4466/first_code_review/pull/1#issuecomment-854151839) directly where those codes are :facepalm: :factory_worker: 
- [ ] I suggest you write a short summary of the review, that summary of the review is the one which gives an overview of the whole review review process, this makes it very simple for the who receives the feedback from reviews to understand what was the review about and what they need to do, and again remember to use polite  way while phrasing your summary. :writing_hand: :pencil2:  :memo: 
- [ ] I would advise you to add some emojis while providing code review suggestions, because that makes review feeling more human :smile: :smiley: